frontend/dist/shotgrid-addon.js

```diff
@@ -122,6 +122,142 @@ const populateTable = async () => {
+  sgAuthToken = await axios
+    .post(`${sgBaseUrl}/auth/access_token`, {
+        client_id: addonSettings.shotgrid_script_name,
+        client_secret: addonSettings.shotgrid_api_key,
+        grant_type: "client_credentials",
+    }, {
+      headers: {
+        'Content-Type': 'application/x-www-form-urlencoded',
+        'Accept': 'application/json'
+      }
+    })
+    .then((result) => result.data.access_token)
+    .catch((error) => {
+      console.log("Unable to Acquire the Shotgrid Authorization Token!")
+      console.log(error)
+    });
+
+    sgUsers = await axios
+      .get(`${sgBaseUrl}/entity/human_users?fields=*`, {
+        headers: {
+            'Authorization': `Bearer ${sgAuthToken}`,
+            'Accept': 'application/json'
+        }
+      })
+      .then((result) => result.data.data)
+      .catch((error) => {
+        console.log("Unable to Fetch Shotgrid Users!")
+        console.log(error)
+      });
+
+    var sgUsersConformed = []
+    
+    users_to_ignore = ["dummy", "root", "support"]
+    if (sgUsers) {
+      sgUsers.forEach((sg_user) => {
+        if (
+          sg_user.attributes.sg_status_list == "act" &&
+          !users_to_ignore.some(item => sg_user.attributes.email.includes(item))
+        ) {
+          sgUsersConformed.push({
+            "login": sg_user.attributes.login,
+            "name": sg_user.attributes.name,
+            "email": sg_user.attributes.email,
+          })
+        }
+      });
+    }
+    return sgUsersConformed;
+}
+
+
+const getAyonUsers = async () => {
+  /* Query AYON for all existing users. */
+  ayonUsers = await axios({
+    url: '/graphql',
+    headers: {"Authorization": `Bearer ${accessToken}`},
+    method: 'post',
+    data: {
+      query: `
+        query ActiveUsers {
+          users {
+            edges {
+              node {
+                attrib {
+                  email
+                  fullName
+                }
+                active
+                name
+              }
+            }
+          }
+        }
+        `
+    }
+  }).then((result) => result.data.data.users.edges);
+
+  var ayonUsersConformed = []
+
+  if (ayonUsers) {
+    ayonUsers.forEach((user) => {
+      ayonUsersConformed.push({
+        "name": user.node.name,
+        "email": user.node.attrib.email,
+        "fullName": user.node.attrib.fullName,
+      })
+    })
+  }
+    return ayonUsersConformed
+}
+
+
+const createNewUserInAyon = async (login, email, name) => {
+  call_result_paragraph = document.getElementById("call-result");
+
+  response = await ayonAPI
+    .put("/api/users/" + login, {
+      "active": true,
+      "attrib": {
+        "fullName": name,
+        "email": email,
+      },
+      "password": login,
+    })
+    .then((result) => result)
+    .catch((error) => {
+      console.log("Unable to create user in AYON!")
+      console.log(error)
+      call_result_paragraph.innerHTML = `Unable to create user in AYON! ${error}`
+  });
+}
```

Comment:
hmm yeah I'm not sure, my syncs still work fine, I just synced a few more differences I had on our fork

Reply:
So some changes needed to be done so we could merge this. Please review following PR on your fork @fabiaserra https://github.com/fabiaserra/ayon-shotgrid/pull/1. This should probably interfere with your user base since we are only normalizing users which are having email in login. Also I had removed the password and active flag since it is active by default. The original SG login was added into `user.data.sg_login`.